hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -106,6 +102,34 @@ protected Object getFallback() {
     return hystrixCommand.execute();
   }
 
+  private HystrixCommand.Setter config(final Object proxy, final Method method) {
```

Comment:
I'm not familiar with javanica..

Not sure I understand your idea... can you elaborate?